pkg/objectupdate/sched/sched_test.go

```diff
@@ -73,10 +73,6 @@ var dpAllOptions = &appsv1.Deployment{
 								Name:  "PFP_STATUS_DUMP",
 								Value: "/run/pfpstatus",
 							},
-							{
```

Comment:
this change is good. I think we do want to preserve the intent of the tests, so check that we can effectively change the setting, using the new way (ConfigParams), don't we?